spec/middleware-to-promise.helper.spec.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+describe("middleware-to-promise.helper", () => {
+    var factory, middlewareTest, mockCall;
+
+    require("../lib/promises.helper");
+    require("../lib/fail.helper");
+    beforeEach(() => {
+        var second;
+
+        require("../lib/middleware-to-promise.helper");
+
+        mockCall = jasmine.createSpyObj("mock", [
+            "call",
+            "otherCall"
+        ]);
+        mockCall.call.andCallFake(() => {
+            return new Promise((resolve) => {
+                return resolve();
+            });
+        });
+        mockCall.otherCall.andCallFake(() => {
+            return new Promise((resolve, reject) => {
+                return reject();
+            });
+        });
+        factory = (secondResult) => {
+            second = createSpyObj("afsdf", [
```

Comment:
Do you need jasmine. to precede createSpyObj?

Is there a reason this is "afsdf"? More generally, is there a naming convention for the createSpyObj?